### DISCUSSION TITLE
PGTK school is not presented when selecting schools in github student developer pack flow

### DISCUSSION THREAD

**Discussion by bhubavenski** [community/community#139634] [GitHub Education]
### Select Topic Area

Question

### Body

Hello! I am trying to sign in for the github student developer pack, but when I come to pick my school, its not presented in the list. It's name is PG in telecommunications is Student City, Sofia, Bulgaria. Please add it to the list

**Comment by ebndev** [comment]
Hi @bockster6669, thanks for posting in GitHub Discussions! 

I've gone ahead and moved this post to our `GitHub Education` category, as this topic may be more relevant there and will ensure it gets seen by the proper teams.

**Comment by Injamul2k24** [comment]
Hi @bockster6669 ,
 contact or mail  github campas expert  support team. They may resolve this
